### PR TITLE
Adjust for release/12.x branch of LLVM

### PR DESCRIPTION
**Pull request by AlexeySotkin**
Signed-off-by: Alexey Sotkin <alexey.sotkin@intel.com>

### CODE REVIEW

**Comment by AlexeySotkin** [comment]
This PR should address (at least partially) #914

**Review by svenvh — COMMENTED** [review]
The changes look good to me, but please make sure you merge it to the correct branch. The PR currently says "AlexeySotkin wants to merge 1 commit into KhronosGroup:master ..."

**Comment by AlexeySotkin** [comment]
> The changes look good to me, but please make sure you merge it to the correct branch. The PR currently says "AlexeySotkin wants to merge 1 commit into KhronosGroup:master ..."

Thanks for catching this!

**Comment by AlexeySotkin** [comment]
I have restarted GitHub Action builds after the rebase.

File: .travis.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ compiler:
 env:
   global:
     - MAKEFLAGS="-j2"
-    - LLVM_VERSION=13
+    - LLVM_VERSION=12
```
Comment (svenvh) [inline .travis.yml:46]: Looking at the travis failure, it seems like you also need to update "llvm-toolchain-bionic" to "llvm-toolchain-bionic-12" on line 28 above.

Comment (AlexeySotkin) [reply .travis.yml:46]: Right, thanks! I have updated the patch.

**Review by AlexeySachkov — APPROVED** [review]

**Review by svenvh — APPROVED** [review]

**Pull request merged**